velox/dwio/parquet/tests/reader/E2EFilterTest.cpp

```diff
@@ -256,6 +256,18 @@ TEST_F(E2EFilterTest, integerDictionary) {
       20);
 }
 
+TEST_F(E2EFilterTest, timestampDictionary) {
+  options_.dataPageSize = 4 * 1024;
+  options_.writeInt96AsTimestamp = true;
+
+  testWithTypes(
+      "timestamp_val:timestamp",
```

Comment:
Let's one more timestamp field so that we can test more cases like sparse row set

Reply:
Added one more timestamp field. Do I need extra change to test sparse row set?